### ISSUE TITLE
Disable right click on <amp-mathml>

### ISSUE THREAD

**Issue by fstanis** [ampproject/amphtml#17039]
## What's the issue?

It seems that the formulas produced by MathJax in `<amp-mathml>` have a right-click menu, which is inconvenient for two reasons:

- The iframe containing it is too small, so the menu is not visible.
- Right-clicking and creating the menu seems to change the layout and add some padding, making part of the formula invisible.

[Per the docs](http://docs.mathjax.org/en/latest/options/hub.html), seems we just need to set the `showMathMenu` option to `false`, but I'm not sure how to correctly do this with the current setup, since we're using a built-in config.

## How do we reproduce the issue?

Open any AMP page with `<amp-mathml>` on a PC and right click anywhere in the formula.

## Which AMP version is affected?

Tested with 1531800879103.
